src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String t){
+        description = t;
+        isDone = false;
+    }
+
+    @Override
+    public String toString(){
+        return ("[T][" + getStatusIcon() + "] " + getDescription() + "\n");
+    }
+
+    public String getStatusIcon(){
+        return isDone()? "X":" ";
```

Comment:
Consider standardizing the spaces before and after operators, e.g isDone() ? "X" : " "